components/common/accordion/accordion.scss

```diff
@@ -0,0 +1,61 @@
+.Accordion {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
+    .accordion-item {
+        border: 1px solid #ccc;
+
+        .accordion-item:not(:first-of-type) {
+          border-top: 0;
+      }
+  
+      .accordion-item-header {
+          display: flex;
+          align-items: flex-start;
+          width: 100%;
+          background-color: #fff;
+          border: 0;
+          padding: 15px 20px;
+          font-size: 18px;
+          font-weight: 400;
+          flex-direction: column;
+  
+          &__title {
+              width: 100%;
+              margin: 0;
+          }
+  
+          &__description {
+            width: 100%;
+            margin: 0;
+          }
+  
+          &:after {
+            content: "";
+            flex-shrink: 0;
+            opacity: 0.5;
+            width: 18px;
+            height: 18px;
+            margin-left: auto;
+            background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='%23212529'%3e%3cpath fill-rule='evenodd' d='M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z'/%3e%3c/svg%3e");
+            background-repeat: no-repeat;
+            background-size: 100%;
+            transition: transform 0.2s ease-in-out;
+        }
+      }
+  
+      .accordion-item.active .accordion-item-header::after {
+          transform: rotate(-180deg);
+      }
+  
+      .accordion-item-container {
+          transition: height 0.2s ease-in-out;
+          overflow: hidden;
+          &__content {
+            border-top: 1px solid #cccccc;
```

Comment:
don't forget to remove the border for the first child to prevent duplicate borders